tensorflow/tensorflow.bzl

```diff
@@ -12,7 +12,7 @@ def _parse_bazel_version(bazel_version):
   # Turn "release" into a tuple of integers
   version_tuple = ()
   for number in parts[0].split('.'):
-    version_tuple += (int(number),)
+    version_tuple += (int(number.strip('abcdefghij')),)
```

Comment:
can you replace 'int' with 'str' here ?

version_tuple += (str(number),)

We'll compare strings lexicographically, which will probably solve your problem here.

Ideally we'd all use semantic versioning conventions though.